server/config.ts

```diff
@@ -90,15 +90,15 @@ export default {
         response: Number(get('MANAGE_OFFENCES_API_TIMEOUT_RESPONSE', 10000)),
         deadline: Number(get('MANAGE_OFFENCES_API_TIMEOUT_DEADLINE', 10000)),
       },
-      agent: new AgentConfig(),
+      agent: new AgentConfig(Number(get('MANAGE_OFFENCES_API_TIMEOUT_RESPONSE', 20000))),
```

Comment:
I think this needs to be the same value as line 90 and 91
```suggestion
      agent: new AgentConfig(Number(get('MANAGE_OFFENCES_API_TIMEOUT_RESPONSE', 10000))),
```